core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorSplit.java

```diff
@@ -20,6 +20,13 @@
 
 public interface ConnectorSplit
 {
+    /**
+     * Returns whether a ConnectorSplit can be scheduled on any node.
+     * <p>
+     * When true, the addresses returned by {@link #getAddresses()} are treated by the scheduler
+     * as hints, which it may not follow strictly.
+     * When false, the split will always be scheduled on one of the addresses returned by {@link #getAddresses()}.
```

Comment:
Both these cases are the same, that the split will be scheduled on one of the nodes returned by `getAddresses()`.

Reply:
I've updated the comment now